src/utils/presence.js

```diff
@@ -94,7 +94,7 @@ export const presenceToHumanTime = (presence: UserPresence, status?: UserStatus)
     : `${formatDistanceToNow(lastTimeActive)} ago`;
 };
 
-export const statusFromPresence = (presence?: UserPresence): PresenceStatus => {
```

Comment:
In commit message:

    See
      https://github.com/zulip/zulip-mobile/pull/5224/commits/aa40a6fff157fbc5e298369c09fd5529d727aa7b#r802272836

GitHub unfortunately has a habit of taunting you if you try to follow a link like this, one referring to both a PR and a commit ID:
![image](https://user-images.githubusercontent.com/28173/154359927-d0557626-2065-473a-9dfd-1924598742ff.png)
> :telescope: 
> We went looking everywhere, but couldn’t find those commits.

(I say "taunting" partly because "went looking everywhere" is a bald-faced lie: if you edit the URL to just refer directly to the commit without the PR, GitHub finds the commit with no trouble. Like this: https://github.com/zulip/zulip-mobile/commit/aa40a6fff157fbc5e298369c09fd5529d727aa7b

That doesn't show the discussion *on* the PR, though, which is I think what you mainly meant to link to in this case.)

Instead, if you go to the main PR page and find the subthread there, you get a link like this: `https://github.com/zulip/zulip-mobile/pull/5224#discussion_r802272836` and those are more stable.

(There's another link like this one in a later commit too, "user status types: Say that /register response data is string-keyed".)